src/Core/Analysis/IAnalysisIssue.cs

```diff
@@ -100,4 +100,12 @@ public enum AnalysisIssueType
         Bug,
         Vulnerability
     }
+
+    public static class IAnalysisIssueExtensions
+    {
+        public static bool IsFileLevel(this IAnalysisIssueBase issue)
```

Comment:
We discussed About making this for IAnalysisIssue but to be able to use this in issuevisualisation we need to use base interface.